### PR TITLE
Terraform for developrs blog

### PR DESCRIPTION
**Pull request by AdminTurnedDevOps**
The blog below is for the Terraform for Developers blog, which was agreed upon for writing between myself and John Bristowe.

### CODE REVIEW

**Comment by wordlee** [comment]
That's for submitting, Michael. I made some minor edits to the post. Most of the edits are for flow and consistency with the rest of the blog. I did break the conclusion into a bullet list as the sentence structure was a bit repetitive. Hopefully all the edits make sense. I plan to take another look at the post early next week.

**Comment by wordlee** [comment]
Michael, can you sign the  CLA above, and add your email address to the author section of the yaml header in the index.md file, please.

**Comment by AdminTurnedDevOps** [comment]
Sounds good to me, thank you. I've signed the CLA and the edits look great.


Kind Regards,



*Michael J. Levan*

Founder and Chief Engineer
------------------------------

*CloudDev.Engineering LLC*

*E* – michael.levan@clouddev.engineering

*W* - Website <https://www.clouddev.engineering/>

*T* - @TheNjDevOpsGuy

*G* - GitHub <https://github.com/AdminTurnedDevOps>

**Comment by AdminTurnedDevOps** [comment]
Testing Terraform Blog Post

**Comment by AdminTurnedDevOps** [comment]
@wordlee New PR is in: https://github.com/OctopusDeploy/blog/pull/300

File: blog/2020-06/infrastructure-as-code-with-terraform-for-developers/index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+---
+title: Infrastructure as code with Terraform for developers
+description:
```
Comment (wordlee) [inline blog/2020-06/infrastructure-as-code-with-terraform-for-developers/index.md:3]:
```suggestion
description: Learn about infrastructure as code and Terraform for developers.
```

File: blog/2020-06/infrastructure-as-code-with-terraform-for-developers/index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
+---
+title: Infrastructure as code with Terraform for developers
+description: Learn about infrastructure as code and Terraform for developers.
+author:
```
Comment (wordlee) [inline blog/2020-06/infrastructure-as-code-with-terraform-for-developers/index.md:4]:
```suggestion
author: michael.levan@clouddev.engineering
```

**Pull request merged**